avro/src/main/java/com/fasterxml/jackson/dataformat/avro/schema/AvroSchemaHelper.java

```diff
@@ -71,6 +72,14 @@ public abstract class AvroSchemaHelper
             String.class
     ));
 
+    /**
+     * 
+     * Jackson annotation introspector for verifying enum default annotation for Avro Schema generation
+     * 
+     * @since 2.16
+     */
+    private static final JacksonAnnotationIntrospector JACKSON_ANNOTATION_INTROSPECTOR = new JacksonAnnotationIntrospector();
```

Comment:
No, `AnnotationIntrospector` should be passed from outside; it may be reconfigured by caller.
We should not rely on default instance.

Reply:
makes sense.  thanks